torchvision/datasets/caltech.py

```diff
@@ -18,10 +19,16 @@ class Caltech101(VisionDataset):
     Args:
         root (string): Root directory of dataset where directory
             ``caltech101`` exists or will be saved to if download is set to True.
-        target_type (string or list, optional): Type of target to use, ``category`` or
-            ``annotation``. Can also be a list to output a tuple with all specified
-            target types.  ``category`` represents the target class, and
-            ``annotation`` is a list of points from a hand-generated outline.
+        target_type (string or list, optional): Type of target to use, ``category``, ``annotation``, ``box_coord``, or
```

Comment:
I can't really think of a good reason to use `box_coord` instead of `bbox`. Don't we use `bbox` everywhere else?

I know we are "using what the original dataset defines" but that's.... probably a bad idea? Doesn't it just lead to more and more inconsistencies? We don't have to keep doing this right?